package.json

```diff
@@ -44,6 +44,11 @@
         "title": "Clean Build",
         "category": "Swift"
       },
+      {
+        "command": "swift.quickSwitchTarget",
```

Comment:
Given you changed the function name, you might want to do the same to the command and title.

Reply:
That's my bad, sorry. These should be unified now.